### DISCUSSION TITLE
Vertically center align icons

### DISCUSSION THREAD

**Discussion by scottAnselmo** [jpanther/lynx#56] [Ideas]
Currently on a 1080p screen the icons are noticeably off. Their center point looks to be about 2/3 of the way down, not 1/2. I suggest updating the CSS to better vertically align them. I would submit a PR, but there are arguably a lot of ways to do this with CSS so I defer to folks who are domain experts with this kind of thing

![image](https://user-images.githubusercontent.com/3056597/224862914-879f23f8-b47a-4b80-af36-a83d5cc78615.png)

**Comment by MrHedmad** [comment]
Hello, sorry to necro-bump, but I've stumbled in this while trying to fix the issue myself. 
This is probably bad in a million ways that I don't know about, but I managed to fix it with this CSS snippet:
```css
.link svg {
  display: inline-block;
  vertical-align: middle;
}
```
This is included in `assets/css/custom.css`. Hope this can help other noobs like me before an official fix is released (if ever). A side effect is that it very slightly stretches the buttons vertically.
